jsfuck.js

```diff
@@ -76,16 +76,16 @@
     'Y':   null,
     'Z':   null,
 
-    ' ':   '(NaN+[]["flat"])[11]',
+    ' ':   '(NaN+[]["at"])[11]',
     '!':   null,
     '"':   '("")["fontcolor"]()[12]',
     '#':   null,
     '$':   null,
-    '%':   'Function("return escape")()([]["flat"])[21]',
+    '%':   'Function("return escape")()([]["at"])[19]',
```

Comment:
`19` is super expensive. It would be better if rewritten like this:

`Function("return escape")()([]["at"])[22]` or `Function("return escape")()([]["at"])[13]`